doc/sphinxman/source/detci.rst

```diff
@@ -193,6 +198,32 @@ Basic DETCI Keywords
 For larger computations, additional keywords may be required, as
 described in the DETCI section of the Appendix :ref:`apdx:detci`.
 
+.. index:: 
+   pair: CI; spin multiplicities of higher roots
+
+Spin Multiplicities of Higher Roots
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As mentioned above, DETCI works in a basis of Slater determinants, rather than
+configuration state functions.  The correct value of :math:`M_s` is easily enforced
+by fixing the number of alpha and beta electrons to be constant across all determinants 
+selected for the CI computation.  However, determinant-based codes like DETCI do
+not necessarily enforce the correct spin :math:`S`.  For example, diagonalizing the
+Hamiltonian in a space spanned by determinants with :math:`M_s = 0` can yield 
+both singlet and triplet solutions, because triplets (:math:`S = 1`) also have an
+:math:`M_s = 0` component.  (Indeed, even higher spin multiplicities may be encountered
+if the excitation level is high enough).  For closed-shell references with :math:`M_s = 0`,
+the program will assume by default that a singlet (:math:`S = 0`) computation is desired,
+and will utilize alpha/beta interchange symmetries to speed up the computation 
+(this is controlled by the advanced keyword |detci__ms0|, which defaults to ``TRUE``).
+Thus, if a user requests multiple roots (|detci__num_roots| = :math:`n`), the program
+will typically return singlets and not triplets.  However, if enough roots are sought,
+higher-multiplicities may enter in.  This can be avoided by ensuring that all the guess
+vectors have the correct spin multiplicity, by setting |detci__calc_s_squared| to ``TRUE``).
+For open-shell systems, the |detci__ms0| keyword is typically not relevant, and there
```

Comment:
```suggestion
It is also possible to ask DETCI to compute roots of higher multiplicities.  The desired value of spin may be
provided by setting |detci__S| = :math:`S`.  Typically, it would be a good idea in this context to also set
|detci__calc_s_squared| to ``TRUE`` to ensure the guess roots have the right spin.  However, be advised that
seeking these higher roots may cause convergence problems, because roundoff may allow the lower-lying
roots of lower multiplicities to re-enter the computation (reducing convergence criteria may help).

For open-shell systems, the |detci__ms0| keyword is typically not relevant, and there
```